src/mmap.rs

```diff
@@ -130,15 +130,16 @@ impl<B: Bitmap> GuestRegionMmap<B> {
     }
 }
 
+#[cfg(not(all(feature = "xen", unix)))]
 impl<B: NewBitmap> GuestRegionMmap<B> {
     /// Create a new memory-mapped memory region from guest's physical memory, size and file.
     pub fn from_range(
         addr: GuestAddress,
         size: usize,
         file: Option<FileOffset>,
     ) -> result::Result<Self, Error> {
-        let region = if let Some(ref f_off) = file {
-            MmapRegion::from_file(f_off.clone(), size)
+        let region = if let Some(ref f_offset) = file {
+            MmapRegion::from_file(f_offset.clone(), size)
```

Comment:
This change is not needed, you can leave it now, but in the future it would be great if we limit the number of changes to make the PRs easier to review. This is especially useful when we have PR with many lines of code like this one.